src/content/basics/giant-swarm-platform-access/index.md

```diff
@@ -0,0 +1,71 @@
++++
+title = "The Giant Swarm Platform Access Routes"
+description = "General Giant Swarm Platform Access Routes Overview"
+date = "2019-05-08"
+weight = 20
+type = "page"
+categories = ["basics"]
++++
+
+# Giant Swarm Platform Access Routes
+
+## Intro
+Access to Giant Swarm platforms can be split into two parts. 
+
+1. Admin Access - designated for the Giant Swarm employees for managing/development/support purposes.
+
+2. User Access - designated for GiantSwarm customers to interact with the offered services.
+
+Both are managed via Giant Swarm VPN hopp that enables connection through jumphost of the cluster.
```

Comment:
```suggestion
Both are managed via Giant Swarm Virtual Private Network (VPN) hopp that enables connection through jumphost of the cluster.
```

When introducing an abbreviation/acronym, we use the long form and set the short form in brackets. After that, the short form can be used.